style-guide/styles/wds-components/_community-header.scss

```diff
@@ -0,0 +1,132 @@
+.wds-community-header {
+	-moz-osx-font-smoothing: grayscale;
+	-webkit-font-smoothing: antialiased;
+	background-position: right center;
+	background-repeat: no-repeat;
+	background-size: auto 100%;
+	height: 105px;
+	position: relative;
+	z-index: 1;
+
+	@media #{$wds-breakpoint-l-down} {
+		height: 115px;
+	}
+
+	&::before {
+		content: '';
+		display: block;
+		height: 100%;
+		position: absolute;
+		width: 100%;
+		z-index: -1;
+	}
+
+	&__wordmark {
+		float: left;
+		height: 65px;
+		margin: 20px 0 20px 20px;
+		max-width: 250px;
+		width: auto;
+
+		@media #{$wds-breakpoint-l-down} {
+			height: 49px;
+			margin: 20px 0 0 20px;
+			max-width: 175px;
+			width: auto;
+
+			img {
+				max-height: 49px;
```

Comment:
duplicated `49px`
Don't we want to apply all styles to `img` instead of `__wordmark` wrapper?